frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -789,7 +856,7 @@ export const UpdateInProgress: React.FC<UpdateInProgressProps> = ({
         <NodesUpdatesGroup
           desiredVersion={desiredVersion}
           machineConfigPool={masterMachinePoolConfig}
-          name="Master"
+          name={NodeTypeNames.Master}
```

Comment:
These (this one and the Worker one below) are not translated. They appear to be used directly in both data and text rendered to the screen in `NodesUpdatesGroup`. We should consider splitting them up and translating the visual one and keeping the data one separate.